Archiving/build.gradle

```diff
@@ -18,6 +18,7 @@ allprojects {
         jcenter()
         repositories {
             jcenter()
+            mavenCentral()
             maven { url 'https://maven.google.com' }
             maven { url 'https://tokbox.bintray.com/maven' }
         }
```

Comment:
bintray is being sunset so users won't be able to download artifacts from there anymore.  Should bintray links still be included?